source/slang/slang-ir-insts.h

```diff
@@ -3814,6 +3851,13 @@ struct IRBuilder
 
     IRInst* emitMakeTuple(IRType* type, List<IRInst*> const& args)
     {
+        if (args.getCount() == 1)
+        {
+            if (args[0]->getOp() == kIROp_Expand)
+            {
+                return args[0];
+            }
+        }
```

Comment:
Should we also handle the zero number of argument case?

Reply:
Zero args here should automatically translate to a tuple type with 0 elements, and will be translated to void during lowering.